### PR TITLE
feat(demo): Demo site 

### PR DESCRIPTION
**Pull request by hereismass**
Demo site created with example on how to use the library.

Built with webpack.

a few commands added related to the demo site

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            You can press the button to initialize Hotjar which will allow to use the rest of the features.
+          </p>
+
+          <button class="button" id="init-hotjar-button">Init Hotjar</button>
+
+          <div class="alert alert-danger" id="init-hotjar-alert">Hotjar is not initialized</div>
+        </div>
+        <div class="right">
+          <pre class="prettyprint"><code class="language-js">const siteId = 123;
+cont hotjarVersion = 6;
```
Comment (kaapa-s) [inline demo/index.html:52]: Typo here ;)

Comment (hereismass) [reply demo/index.html:52]: Nice catch :)

File: demo/demo.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    Hotjar.identify(userId, {
+      first_name: firstName,
+      color: favoriteColor,
+    });
+
+    setStatusMessage(
+      '#identify-hotjar-alert',
+      'success',
+      `${firstName} identified with ${favoriteColor} as favorite color`,
```
Comment (kaapa-s) [inline demo/demo.js:57]: Is it possible to display the values entered by the user in bold? It's definately not a blocker but a nice to have ;)

Comment (hereismass) [reply demo/demo.js:57]: Good idea

**Pull request merged**